MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+  return grid;
+});
+{% endhighlight %}
+{% endtabs %}
+
+## Event
+
+The `ItemDragging` event is raised while dragging and dropping the item in the `SfListView`. The `ItemDraggingEventArgs` has the following members which provides the information for the `ItemDragging` event:
+
+ * `Action`: Returns the drag `Action` such as start, dragging, and drop.
+ * `Bounds`: Return bounds of drag item when dragging and dropping.
+ * `Handled`: If this member is set to `true`, dragging can be handled. It is applicable only if `Action` is `Dragging`.
+ * `DataItem`: Returns the underlying data of the dragging item. 
+ * `NewIndex`: Returns the item index of the `DataSource.DisplayItems` where dragging item is going to be dropped.
+ * `OldIndex`: Returns the item index of the `DataSource.DisplayItems` where dragging item started. The OldIndex and NewIndex will be same if `Action` is `Start`.
+ * `Position`: Returns the touch position of the drag item from screen coordinates.
+
+## Auto scroll options
+
+### Auto scroll margin
+
+To adjust auto scroll margin, set a value to the `ScrollMargin` property of `AutoScroller` to enable auto-scrolling while dragging. The default value is `15`. Auto-scrolling will be enabled when reaching `ScrollMargin` from view bounds while dragging.
+
+To disable auto-scrolling, set the value to `0` for `ScrollMargin`.
+
+{% tabs %}
+{% highlight c# %}
+this.listView.AutoScroller.ScrollMargin = 20;
+{% endhighlight %}
+{% endtabs %}
+
+### Auto scroll interval
+
+To adjust auto-scroll interval while dragging, set the `Interval` property of `AutoScroller`. The default value is `150 milliseconds`.
+
+{% tabs %}
+{% highlight c# %}
+this.listView.AutoScroller.Interval = new TimeSpan(0, 0, 0, 0, 200);
+{% endhighlight %}
+{% endtabs %}
+
+### Disable outside scroll
+
+To disable auto-scroll when dragging item moves outside the `SfListView` while dragging, set the `AllowOutsideScroll` property of `AutoScroller` to `false`. The default value is `true`.
+
+{% tabs %}
+{% highlight c# %}
+this.listView.AutoScroller.AllowOutsideScroll = false;
+{% endhighlight %}
+{% endtabs %}
+
+## Disable dragging for particular item
+
+To disable dragging for a particular item, handle the `ItemDragging` event based on the conditions of `Action` event argument.
+
+You can cancel the dragging action for a particular item by setting the `Cancel` property of the `ItemDraggingEventArgs`.
+
+{% tabs %}
+{% highlight c# %}
+private void ListView_ItemDragging(object sender, ItemDraggingEventArgs e)
+{
+  // Disable the dragging for 4th item.
+  if (e.Action == DragAction.Start && e.NewIndex == 3)
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Cancel dropping for the dragged item
+
+To cancel dropping for the dragged item, handle the `ItemDragging` event based on the conditions of `Action` event argument. 
+
+You can cancel the dropping action for a particular item by setting the `Cancel` property of the `ItemDraggingEventArgs`.
```

Comment:
You can cancel the dropping action for an item by setting the `Cancel` property of the `ItemDraggingEventArgs`.

Reply:
Addressed the concerns provided.